examples/siro_sandbox/sandbox_app.py

```diff
@@ -739,23 +749,26 @@ def _free_camera_lookat_control(self):
         )
 
     def _save_recorded_keyframes_to_file(self):
+        if not self._save_filepath_base or not self._recording_keyframes:
```

Comment:
Alternately, you could just avoid calling this if `self._save_filepath_base` is False.

I do think the check for `not self._recording_keyframes` is more like basic error-checking (don't want to save an empty file), so no reason to call the function `check_` just because you're doing that check.

Reply:
:white_check_mark: